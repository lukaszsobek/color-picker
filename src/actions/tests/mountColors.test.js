import { mountColors } from "../";

describe("mountColors",() => {

    it("with undefined availableColors",() => {
        expect(mountColors()).toMatchSnapshot();
    });

    it("with 2 availableColors",() => {

        const availableColors = [1,2]
        expect(mountColors(availableColors)).toMatchSnapshot();
    });
});
