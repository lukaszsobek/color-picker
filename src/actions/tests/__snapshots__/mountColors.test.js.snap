// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mountColors with 2 availableColors 1`] = `
Object {
  "availableColors": Array [
    1,
    2,
  ],
  "type": "MOUNT_COLORS",
}
`;

exports[`mountColors with undefined availableColors 1`] = `
Object {
  "availableColors": undefined,
  "type": "MOUNT_COLORS",
}
`;
